src/view/UserphraseView.cpp

```diff
@@ -17,7 +17,6 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-#include "UserphraseModel.h"
```

Comment:
I deleted it at first because I thought that code is mine(forgot to use git diff for check),
also, the code can execution well in spite of the deletion of the include.
And I think we don't need unnecessary include so I keep the deletion.

Or I shouldn't change the previous code when I cooperate with others in a project?